embedded/embedded-test/src/test/java/org/mule/runtime/module/embedded/ApplicationConfigurationTestCase.java

```diff
@@ -67,6 +67,7 @@
   private static final String LOGGING_FILE = "app.log";
 
   private static final String LISTENER_URL = "http://localhost:%d/test";
+  private static final String DESCRIPTOR_RELATIVE_PATH = "META-INF/mule-artifact/mule-artifact.json";
```

Comment:
There is already a constant for this ArtifactDescriptor#MULE_ARTIFACT_JSON_DESCRIPTOR_LOCATION